client/src/services/local-storage.js

```diff
@@ -0,0 +1,24 @@
+import config from "../config";
+
+export const loadStateFromLocalStorage = () => {
+        const serializedState = localStorage.getItem(config.stateKeyName);
+        if (serializedState === null) {
```

Comment:
You can use this expression to make this code nicer looking (the `&&` has a professional name in javascript, but I forgot it)
```js
serializedState && return JSON.parse(serializedState)
```
Remove the if, try, and catch

Reply:
why remove the try and catch?

Reply:
Sorry, remove only the if and catch:
```js
if (serializedState) {
    try {
        return JSON.parse(serializedState)
    }
}

Reply:
I don't think you can to `try` without `catch`.

Reply:
Yes of course... I meant
```js
if (serializedState) {
    try {
        return JSON.parse(serializedState)
    } catch {}
}